packages/legend-graph/src/graphManager/action/SourceInformationHelper.ts

```diff
@@ -17,7 +17,7 @@
 import { assertTrue } from '@finos/legend-shared';
 import type { SourceInformation } from './SourceInformation';
 
-const COORDINATE_DELIMITER = '__';
+const COORDINATE_DELIMITER = '@';
```

Comment:
Please add a comment here to say 2 things:
1. Explain why we do this: because Pure identifier does not contain the character `@`
2. Create a `TODO`: We should handle quoted identifier case, e.g. `model::something:'identifier with @'`

Reply:
So let's do this

```js
// NOTE: @ is chosen because it is not part of identifier token in Pure grammar
// TODO: handle the case of quote identifier e.g. model::something::'I have an @ in me'
```